workflow-service-sdk/src/main/java/com/redhat/parodos/sdk/model/ArgumentRequestDTO.java

```diff
@@ -19,16 +19,32 @@
 import com.google.gson.annotations.SerializedName;
 import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonWriter;
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
 import java.io.IOException;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.TypeAdapterFactory;
+import com.google.gson.reflect.TypeToken;
+
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import com.redhat.parodos.sdk.invoker.JSON;
+
 /**
  * ArgumentRequestDTO
  */
-@lombok.Data
-@lombok.AllArgsConstructor
-@lombok.Builder
```

Comment:
Is it intentional? Removing all lombok annotations?

Reply:
This is auto generated code, it was removed after changing the generator version

Reply:
I see it was removed explicitly from here:

https://github.com/parodos-dev/parodos/pull/289/files#diff-2fdbd8c157a3ebaa9bccc9812c554feeb04a5cd3b70f8b3222f63cdb3ad6d4dcL156-L159

Wouldn't it work if this one is restrored?

Reply:
To be exact - this is the change in the existing code base which is removed in this PR:
https://github.com/parodos-dev/parodos/blob/main/workflow-service-sdk/pom.xml#L156

Reply:
Lombok's annotations were explicitly added to have builder, getters, setters, and constructor in the generated classes. This eases the interaction with the SDK classes. Can we keep them?